### DISCUSSION TITLE
Nextjs 13 Upgrade working ?

### DISCUSSION THREAD

**Discussion by slamer59** [transitive-bullshit/nextjs-notion-starter-kit#483] [General]
Hello,
I used this starter kit and I build upon it. 
Then try to upgrade.

It works fine in **dev** mode but not in production mode. 

I don't see anything special in client log...

![WhatsApp Image 2023-03-31 at 15 49 01](https://user-images.githubusercontent.com/25283272/229184024-3e562136-55b6-494a-858d-f77b02770727.jpeg)

Does anyone experience this ? 

PS : I don't know where to begin to debug in production mode in nextjs ?

**Comment by eliasfloreteng** [comment]
I have the same issue, it seems to be when deploying on Vercel for me. It works locally on my machine
